galley/pkg/crd/validation/validation.go

```diff
@@ -25,8 +27,12 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pkg/cmd"
 	"istio.io/istio/pkg/kube"
+
+	"istio.io/istio/pkg/probe"
 )
 
+var validationProbe *probe.Probe
```

Comment:
Move this into the `RunValidation` function scope (~line 76)

Reply:
done